src/main/java/junit/extensions/TestDecorator.java

```diff
@@ -11,7 +11,7 @@
  */
 @SuppressWarnings("deprecation")
 public class TestDecorator extends Assert implements Test {
-    protected Test fTest;
+    private Test fTest;
```

Comment:
This change could break users' code (for example, subclasses that reference this field)